luaparse.js

```diff
@@ -889,6 +889,52 @@
     };
   }
 
+  function scanLuaJITImaginaryUnitSuffix() {
```

Comment:
Nit: other `scan*` function return AST nodes. This one merely a partial token. The prefix should have been `read`.